cogs5e/models/homebrew/bestiary.py

```diff
@@ -362,33 +372,73 @@ def parse_critterdb_traits(data, key):
 
         if overrides:
             for override in overrides:
-                attacks.append({'name': override.group(1) or name,
-                                'attackBonus': override.group(2) or None, 'damage': override.group(3) or None,
-                                'details': desc})
+                if override.group('simple'):
+                    attacks.append(Attack.from_dict({
+                        'name': override.group(2) or name,
+                        'attackBonus': override.group(3) or None,
+                        'damage': override.group(4) or None,
```

Comment:
With the new regex groups, check to make sure the group numbers haven't changed

Reply:
They did, but these numbers are the correct ones.